test/extended/testdata/router/router-grpc-interop.yaml

```diff
@@ -134,11 +137,22 @@ objects:
         value: "99"
       - name: GRPC_GO_LOG_SEVERITY_LEVEL
         value: "info"
+      readinessProbe:
+        httpGet:
+          path: /healthz
+          port: 8080
+        initialDelaySeconds: 10
+        periodSeconds: 3
+      env:
+      - name: GODEBUG
+        value: http2debug=1
```

Comment:
You define `env` twice; I think this definition needs to be merged into the one above:  https://github.com/openshift/origin/blob/d9b8b7fa56c3a4d36584d6055de5a2d602b37d3e/test/extended/testdata/router/router-grpc-interop.yaml#L135-L139

Reply:
Thanks. Done.